examples/browser/wdio.conf.js

```diff
@@ -41,7 +46,7 @@ exports.config = {
     // and 30 processes will get spawned. The property handles how many capabilities
     // from the same test should run tests.
     //
-    maxInstances: 10,
+    maxInstances: debug ? 1 : 10,
```

Comment:
I wonder in `capabilities` below `maxInstances` is 5, what does it mean?

Reply:
I think one is for the amount of browsers running tests per browser type, and the other is for the number of different types of browser running? i.e firefox/chrome/i.e

Reply:
so it means that when you run for chrome, it is not taken into the account? I meant should not you do the same in `capabilities`